README.md

```diff
@@ -90,3 +90,15 @@ To make it permanent add this line to your `~/.zshrc`.
 ### Documentation
  - [OpenCompose file reference documentation](https://github.com/redhat-developer/opencompose/blob/master/docs/file-reference.md)
  - You can find more in [doc/](https://github.com/redhat-developer/opencompose/tree/master/docs) folder
+
+### Contact
+We always welcome your feedback and thoughts on the project! Come and join us on [Slack](https://opencompose.slack.com) and join our [mailing list](https://groups.google.com/forum/#!forum/opencompose).
+
+#### Slack
+[opencompose.slack.com](https://opencompose.slack.com)
```

Comment:
I'm not sure about slack either :-(

1) It is not working as you described ;-) Users need invitation first to sign up to opencompose slack.

2) Isn't ML enough for now?